iso15118/secc/comm_session_handler.py

```diff
@@ -286,6 +286,9 @@ async def get_from_rcv_queue(self, queue: asyncio.Queue):
                         )
                     )
                     self.comm_sessions[notification.ip_address] = (comm_session, task)
+                    await self.evse_controller.set_status_secc_session(
+                        ServiceStatus.STARTING
+                    )
```

Comment:
We currently use ServiceStatus to indicate starting and stopping of the entire iso15118_service in our pro version. I see the intention here is to indicate starting and stopping of charging sessions instead. Could you add a new enum SessionStatus along the lines of SESSION_STARTED, SESSION_ENDED for this, please?

Reply:
I've checked if this status is relevant in our code. But it is not. We can remove the whole function.